gooddata-dbt/gooddata_dbt/dbt/profiles.py

```diff
@@ -180,6 +180,9 @@ def profile(self) -> DbtProfile:
     def target(self) -> DbtOutput:
         for output in self.profile.outputs:
             if output.name == self.args.target:
+                if self.args.gooddata_upper_case:
```

Comment:
Nitpick: consider mering these to condition.

Reply:
IMO I cannot do it, there is a line `return output` outside the inner IF.

Reply:
Overlooked that. Thanks.